src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -2400,6 +2400,58 @@ def test_webapp_vnetDelegation(self, resource_group):
         ])
 
 
+class WebappDeploymentLogsScenarioTest(LocalContextScenarioTest):
+    @ResourceGroupPreparer()
+    def test_webapp_show_deployment_logs(self, resource_group):
```

Comment:
@calvinsID this doesn't test for functionapp can we update an existing function app E2E test to test show deployment

Reply:
Added!